specs/0111-redesign-profile-page.md

```diff
@@ -0,0 +1,113 @@
+
+Example: if a person has 100 approved translations, but 60 are self-approved, the self-approval ratio will be 60%.
+
+The graph displays both the selected month’s ratio and the average for the previous 12 months.
+
+## Contribution graph
+
+The contribution graph displays data over time about the following actions:
+* `Submissions`: translations submitted, including both suggestions and direct translations.
+* `Reviews performed`: reviews performed on pending suggestions.
+* `Reviews received`: reviews received from another translator.
+* `Submissions and reviews`: submissions + reviews performed.
+* `All activities`: submissions + reviews performed + reviews received.
+
+Only one data point is displayed at a time, and it can be selected using the dropdown list above the graph. The default view is `All activities`, since it provides a good picture of the user’s activity in Pontoon.
+
+Each square in the graph represents a day, with color changing depending on the amount of contributions. It’s possible to click on a column to select a specific week, or change the year currently displayed using the links in the rightmost column.
+
+When selecting a specific time range, the data displayed below the graph is updated accordingly. This doesn’t affect the approval and self-approval graphs.
+
+## Other data in the sidebar
+
+The sidebar displays contact information for the contributor:
+* Name.
+* Nickname.
+* Description.
+* Contact email address.
+* External accounts.
+* Last login.
+* Latest activity.
+* Join date (header `Member since`).
+* Translation rights (header `Translator`), followed by a list of locales for which the user has translator rights. The section is hidden if the user doesn’t have translator rights.
+* Locale manager rights (header `Team  Manager`), followed by a list of locales for which the user has manager rights.
+* A flag indicating if the user is an administrator. The section is hidden if the user doesn’t have manager rights.
+
+## Visibility of data
+
+Most of the data will be visible by default to all users, including non authenticated visitors (i.e. users not logged in to Pontoon).
+
+Visibility for the following fields can be limited by user settings:
+* Email address.
+* External accounts.
+* Self-approval ratio.
+* Approval ratio.
+
+Limited means that non authenticated users, and other contributors, won’t be able to see this information when they visit the profile page. But this information will be always visible to users with translator rights for any locale in Pontoon (translators, locale managers, administrators).
+
+## Dependencies and other changes
+
+### Contact email address
+
+In order to support this new profile, users should be able to provide a contact email address:
+* This is the address that will be displayed in the profile page (pending personal visibility settings), and can be different from the one used to log in to Pontoon.
```

Comment:
"the one used to log in to Pontoon", which will be used by default.

Reply:
That wasn't the intent behind the current spec though.

The email used to log in won't be displayed by default. We display an email only if the user provides a different email, specifically for that purpose. But I realize this would allow users to have no email displayed, which makes it impossible for managers to contact them, unless we display both login and contact email addresses.